news/_posts/2021-03-02-archived-modules.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Latest news about modules"
```

Comment:
Can we find a better name, as in 2 years this article title will be not changed 😅  ? Also a good title helps SEO and helps people to find it when using search.

Suggestions:
- "Curating the list of PrestaShop open source modules"
- "Archiving some of our modules"
- "Focusing on essentials modules"

I think the title could help the reader by giving the main idea 😉

Reply:
Yes, good idea :+1: